### ISSUE TITLE
Base Unit for Bitcoin?

### ISSUE THREAD

**Issue by hariesnurikhwan** [nikolaeu/Numi-extensions#65]
I was looking to add satoshi unit but unfortunately there's no base unit for bitcoin. If I'm not mistaken there's unit for Bitcoin in Numi.

**Comment by nikolaeu** [comment]
Yeah, it's impossible for now to add satoshi, since base unit for currency is USD now (and satoshi should be change according to BTC rate). Will think about workaround for this.

**Comment by 2ge** [comment]
I added it. it is BTC. Feel free to use this:

satoshi.js:
```
numi.addUnit({
    "id": "satoshi",
    "phrases": "satoshi, satoshis, sato, sat, bit",
    "baseUnitId": "BTC",
    "format" : "sat",
    "ratio" : 0.00000001,
});
```
